tad/utils/path.py

```diff
@@ -0,0 +1,12 @@
+from pathlib import Path
+
+
+def is_safe_path(basedir: Path, path: Path, follow_symlinks: bool = True) -> bool:
```

Comment:
Nitty picking: this is nice, but I don't see it being used except for test coverage.

Reply:
It was used, but got removed because we dont use the name anymore in the system file.